### PR TITLE
Cleanup ~40 warnings & unused variables

### PR DESCRIPTION
**Pull request by suirad**

### CODE REVIEW

**Comment by Maxopoly** [comment]
ok to test

**Comment by Maxopoly** [comment]
ok to test

**Comment by suirad** [comment]
Added another commit, didn't squash to keep it separate from all the other changes in the the other commit. Its a fix for #95.

**Comment by ttk2** [comment]
@rourke750  @erocs  this a good idea?

**Comment by rourke750** [comment]
Doesn't hurt.  I'll look later though.  Super busy week this week.

File: src/vg/civcraft/mc/citadel/PlayerState.java [hunk 1]
```diff
@@ -85,7 +82,7 @@ public void setMode(ReinforcementMode mode) {
      * @param ItemStack.
      */
     public void setFortificationItemStack(ItemStack fortificationStack) {
-        this.fortificationMaterial = fortificationStack.getType();
+        fortificationStack.getType();
```
Comment (rourke750) [inline src/vg/civcraft/mc/citadel/PlayerState.java]: ???

File: src/vg/civcraft/mc/citadel/database/CitadelReinforcementData.java [hunk 2]
```diff
@@ -447,7 +449,7 @@ else if (rein instanceof NaturalReinforcement){
 			String world = loc.getWorld().getName();
 			Material mat = rein.getMaterial();
 			int dur = rein.getDurability();
-			String chunk_id = loc.getChunk().toString();
+			loc.getChunk().toString();
```
Comment (rourke750) [inline src/vg/civcraft/mc/citadel/database/CitadelReinforcementData.java]: ???

File: src/vg/civcraft/mc/citadel/database/CitadelReinforcementData.java [hunk 3]
```diff
@@ -481,7 +483,7 @@ else if (rein instanceof NaturalReinforcement){
 		}
 		else if (rein instanceof MultiBlockReinforcement){
 			MultiBlockReinforcement mbRein = (MultiBlockReinforcement) rein;
-			Location loc = mbRein.getLocation();
+			mbRein.getLocation();
```
Comment (rourke750) [inline src/vg/civcraft/mc/citadel/database/CitadelReinforcementData.java]: ???

File: src/vg/civcraft/mc/citadel/reinforcement/PlayerReinforcement.java [hunk 4]
```diff
@@ -167,7 +168,8 @@ public String getStatus() {
     private void checkValid(){
     	if (!g.isValid()){ // incase it was recently merged/ deleted.
     		String group = Citadel.getCitadelDatabase().getSavedGroupName(this);
-    		g = NameAPI.getGroupManager().getGroup(group);
+    		NameAPI.getGroupManager();
```
Comment (rourke750) [inline src/vg/civcraft/mc/citadel/reinforcement/PlayerReinforcement.java]: ???

**Comment by rourke750** [comment]
ok to test

**Pull request closed without merging**